<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reservation</class>
 <widget class="QDialog" name="reservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pas d'erreurs pour l'instant :)</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>578</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>idGroupe</string>
         </property>
        </widget>
        <widget class="QComboBox" name="idGrp"/>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Début</string>
         </property>
        </widget>
        <widget class="QTimeEdit" name="hDeb">
         <property name="maximumDateTime">
          <datetime>
           <hour>20</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="minimumDateTime">
          <datetime>
           <hour>8</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>HH:00</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="date"/>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>idSalle</string>
         </property>
        </widget>
        <widget class="QComboBox" name="salle"/>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="createResa">
        <property name="text">
         <string>Réserver</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QSplitter" name="splitter_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>idResa</string>
         </property>
        </widget>
        <widget class="QComboBox" name="idResa"/>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QSplitter" name="splitter_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QSplitter" name="splitter_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Début</string>
         </property>
        </widget>
        <widget class="QTimeEdit" name="hDeb_2">
         <property name="maximumDateTime">
          <datetime>
           <hour>20</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="minimumDateTime">
          <datetime>
           <hour>8</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>HH:00</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QSplitter" name="splitter_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="date_2"/>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QSplitter" name="splitter_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="modifyResa">
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="dropResa">
        <property name="text">
         <string>Drop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
